main.go

```diff
@@ -35,18 +35,24 @@ const (
 func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	// Only used for debugging.
+
+	// Logging setup
```

Comment:
Please remove all blank likes and comments you added. They are redundant and we do not use empty lines inside functions.